Source/DataSources/EntityCluster.js

```diff
@@ -527,43 +529,55 @@ define([
 	}
     });
 
+    function createGetEntity(collectionProperty, CollectionConstructor, unusedIndicesProperty, entityIndexProperty) {
+        return function(entity) {
+            var context = this;
+            var collection = context[collectionProperty];
+            var entityIndices = context._collectionIndicesByEntity[entity.id];
+
+            if (!defined(entityIndices)) {
+                entityIndices = context._collectionIndicesByEntity[entity.id] = Object.create(null);
```

Comment:
Since we know for a fact that this object will have three properties, pointIndex, labelIndex, and billboardIndex, we should specify an object with those three properties here.  It will let the browser make additional optimizations that it otherwise couldn't do.